### ISSUE TITLE
[Bug] Trucking Haul Not Appearing 

### ISSUE THREAD

**Issue by hhenry12** [FrenzyRoleplay/server-issues#253]
**Describe the bug**
While doing the Trucker job(18 wheeler) there have been two cases where the haul I'm supposed to deliver back did not disappear when I got to the destination. The first time, the haul appear after a few seconds but the second when I got to the destination, the haul never appeared. I drove away from the location to see if the issue was the fact the haul never loaded, I thought driving away and driving would load it but that did not work, even after attempting that this twice.

**To Reproduce**
1) Get the trucker job from the Russian
2) Drive to the destination 
3) The haul does not show or appear when you get to the location.

**Expected behavior**
When you arrived at the location to pick up the load, the haul should appear.

**Screenshots**
[The Screenshot](https://i.imgur.com/caypYdp.jpg)

**Additional context**
Please note that I did not do anything to my folder, such as clear cache prior to the server wipe or after.

**Comment by laythdv** [comment]
Just press "E" to spawn the truck. The truck doesn't automatically spawn until YOU spawn it yourself by pressing E.

**Comment by hhenry12** [comment]
> Just press "E" to spawn the truck. The truck doesn't automatically spawn until YOU spawn it yourself by pressing E.

I try that as well and it still wouldn't spawn. I ran around the area spamming my "E" and it still wouldn't spawn. Also when you say "truck" you mean you mean the "haul" but not the actual truck". Correct?

**Comment by laythdv** [comment]
Yes - the haul. If this happens again, see if you can grab a video instead.

**Comment by badcodestv** [comment]
Yes. The haul. Try to grab a video next time. I'll try to mess with this over the next day or so

**Comment by hhenry12** [comment]
> Yes. The haul. Try to grab a video next time. I'll try to mess with this over the next day or so

Will do.
